src/modal/Cart.java

```diff
@@ -0,0 +1,78 @@
+package modal;
+
+import java.util.ArrayList;
+
+public class Cart {
+
+    private final static double TAX_PERCENTAGE = 0.20;
+    private ArrayList<CartItem> cartItems;
+    private double deliveryFee;
+
+    public Cart() {
+        this.cartItems = new ArrayList<>();
```

Comment:
small remark, but if you're going to only initialize a collection like this, you can do it on instance level while declaring the variable and use the implicit constructor.
also remove the .this :D